components/banner.tsx

```diff
@@ -1,8 +1,10 @@
+"use client";
```

Comment:
These needed to become client components because they use `usePathname`, which is only available on the client side in the `app` router. It's an annoying next thing.